.gitignore

```diff
@@ -44,6 +44,7 @@
 /test-driver
 /tmp
 /tor-bridges.tor-bridges-ip-port.csv
+/venv
```

Comment:
Adding to `.gitignore` the directory used for the `virtualenv` consistently with `./script/autoapi` docs.